lib/index.ts

```diff
@@ -141,6 +141,10 @@ export interface ExtendedWebDriver extends WebDriver {
 
   // See https://github.com/webdriverio/webdriverio/blob/v4.6.1/lib/protocol/shake
   shakeDevice: () => wdpromise.Promise<void>;
+
+  sendCommand: (cmd: string, params: Object) => wdpromise.Promise<void>;
```

Comment:
We don't have a consistent pattern for naming, but what about something like `sendChromiumCommand` to make the connection to the chromium tools clearer?

Reply:
Done.